### PR TITLE
Delete all button style + fix dialog bug

### PR DESCRIPTION
**Pull request by dieie32**
- delete all button style
- fix dialog bug

### CODE REVIEW

File: src/redux/news/news.operations.js [hunk 1]
```diff
@@ -164,41 +164,62 @@ const createArticle = async (news) => {
...
-            author {
-              name {
-                value
-              }
-            }
-          }
-          ... on Error {
-            message
-            statusCode
+  // const result = await client.mutate({
```
Comment (moran711) [inline src/redux/news/news.operations.js:167]: Why you commented this line?

File: src/redux/news/news.operations.js [hunk 2]
```diff
@@ -164,41 +164,62 @@ const createArticle = async (news) => {
...
+  //         }
+  //       }
+  //     }
+  //   `,
+  //   fetchPolicy: 'no-cache'
+  // });
+  // await client.resetStore();
+  setItems(
+    `mutation($id: ID!, $news: NewsInput!) {
+    updateNews(id: $id, news: $news) {
```
Comment (moran711) [inline src/redux/news/news.operations.js:194]: You can pass here `query` variable with mutation string.

File: src/redux/news/news.operations.js [hunk 3]
```diff
@@ -164,41 +164,62 @@ const createArticle = async (news) => {
...
+      }
+    }
+  }`,
+    {
+      id,
+      news
+    }
+  );
+
+  // if (result.data.updateNews.message) {
```
Comment (moran711) [inline src/redux/news/news.operations.js:214]: Why you commented this line?

File: src/pages/news/news-page/news-page.js [hunk 4]
```diff
@@ -57,7 +57,7 @@ const NewsPage = () => {
       dispatch(closeDialog());
       dispatch(deleteArticle(id));
     };
-    openSuccessSnackbar(removeNews, REMOVE_TITLE, REMOVE_MESSAGE, REMOVE_TITLE);
+    openSuccessSnackbar(removeNews, DELETE_TITLE, REMOVE_MESSAGE, DELETE_TITLE);
```
[the remark this answers is not in the document]
Comment (dieie32) [reply src/pages/news/news-page/news-page.js:60]: openSuccessSnackbar need dialogTitle and buttonTitle
image

**Review by Iryna-Bzdyr — APPROVED** [review]

**Review by nadiyafomenko — APPROVED** [review]

**Pull request merged**